R/z_animintHelpers.R

```diff
@@ -173,13 +173,30 @@ hjust2anchor <- function(hjust){
   }
 }
 
+#' @param params Named list of layer parameters
+#' @return NULL if valid, stops with error if invalid showSelected variant found
+validateShowSelectedParams <- function(params) {
```

Comment:
please change fun name to `error_for_showSelected_variants`

Reply:
actually, on second thought, can you just remove this function and please add the logic to the getLayerParams directly?